_includes/header.html

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-dark fixed-top navbar-expand-lg bg-famnm-blue">
+<nav class="navbar navbar-dark fixed-top navbar-expand-lg bg-famnm-blue w-100">
```

Comment:
Why do we need this to be ```width:100```? Does this actually affect the sizing?

Reply:
We set this to `w-100` explicitly to make it clear that it is a full-width navbar. this ensures that if navbar's defaults change should we upgrade bootstrap, that this won't necessarily break. I'd like to air on the side of explicitly getting behavior rather than expecting default behavior to not change.